src/components/common/icons/Progressing.tsx

```diff
@@ -6,6 +6,7 @@ interface ProgressingProps {
     size?: number;
     fullHeight?: boolean;
     theme?: 'white' | 'default';
+    children?: any
```

Comment:
Can we use specific type rather than any?

Reply:
Done.